src/main/kotlin/org/rust/lang/core/psi/RsPsiFactory.kt

```diff
@@ -334,7 +317,7 @@ class RsPsiFactory(
             ?: error("Failed to create match body from patterns: `$arms`")
     }
 
-    private inline fun <reified T : RsElement> createFromText(code: CharSequence): T? =
+    inline fun <reified T : RsElement> createFromText(code: CharSequence): T? =
```

Comment:
Let's better keep it private and make simple `createMembers` like
```kotlin
    fun createMembers(text: String): RsMembers {
        return createFromText("impl T for S {$text}") ?: error("Failed to create members from text: `$text`")
    }
```
and use it `createTraitMembers`